istio/src/main/java/com/alibaba/nacos/istio/mcp/NacosToMcpResources.java

```diff
@@ -132,36 +143,33 @@ public void run() {
     }
     
     private String convertName(Service service) {
-        if (!Constants.DEFAULT_GROUP.equals(NamingUtils.getGroupName(service.getName()))) {
-            return NamingUtils.getServiceName(service.getName()) + "." + NamingUtils.getGroupName(service.getName())
-                    + "." + service.getNamespaceId();
+        if (!Constants.DEFAULT_GROUP.equals(service.getGroup())) {
+            return service.getName() + "." + service.getGroup();
```

Comment:
return service.getName() + "." + service.getGroup()+ "." + service.getNamespace()

Reply:
Thanks for the reminder, it has been corrected